### PR TITLE
chore: Clean up FCO documentation

### PR DESCRIPTION
**Pull request by kevjumba**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: docs/getting-started/architecture-and-components/registry.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Each Feast deployment has a single feature registry. Feast only supports file-ba
 * `Local`: Used as a local backend for storing the registry during development
 * `S3`: Used as a centralized backend for storing the registry on AWS
 * `GCS`: Used as a centralized backend for storing the registry on GCP
+* `[Alpha] Azure`:
```
Comment (felixwang9817) [inline docs/getting-started/architecture-and-components/registry.md:10]: missing a description here?

File: docs/getting-started/concepts/entity.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ Feast refers to this collection of entities for a feature view as an **entity ke
 
 Entities should be reused across feature views. This helps with discovery of features, since it enables data scientists understand how other teams build features for the entity they are most interested in.
 
-Feast will use the feature view concept to then define how to store groups of features in a low-latency online store.
+Feast will use the feature view concept to then define the schema of groups of features in a low-latency online store.
 
 ### Use case #2: Retrieving features
 
-At _training time_, users control what entities they want to look up, for example corresponding to train / test / validate splits. A user specifies a list of _entity keys + timestamps_ they want to fetch point-in-time correct features for to generate a training dataset.
+At _training time_, users control what entities they want to look up, for example corresponding to train / test / validatation splits. A user specifies a list of _entity keys + timestamps_ they want to fetch [point-in-time](./point-in-time-joins.md) correct features for to generate a training dataset.
```
Comment (felixwang9817) [inline docs/getting-started/concepts/entity.md:31]: validation

**Review by felixwang9817 — COMMENTED** [review]
looks good overall

File: docs/getting-started/concepts/entity.md (hunk 2 above)
Comment (kevjumba) [reply docs/getting-started/concepts/entity.md:31]: fixed

File: docs/getting-started/architecture-and-components/registry.md (hunk 1 above)
Comment (kevjumba) [reply docs/getting-started/architecture-and-components/registry.md:10]: ah must have not saved fixed.

**Review by felixwang9817 — APPROVED** [review]
/lgtm

**Pull request merged**